### PR TITLE
service: irs: Add camera support, split processors and implement ImageTransferProcessor

### PR DESCRIPTION
**Pull request by german77**
Depends on Qt multimedia in externals to compile.  This PR implements and adds infrared camera support via a Qt camera. Specifically the image transfer processor. There are other 5 image processors that need to be implemented to have support in all games.

Since each processor requires significant RE they will be implemented separately. Qt camera support web cameras and virtual cameras like OBS virtual camera. In a future I will add alternatives for camera emulation like using a static image.

Irs example from libnx
image

Image captured running `Night Vision` the ir camera on a joycon takes 4 seconds to take a full resolution image. To lower the delay the game lowers the resolution to just 40x30 pixels resulting in a bad quality image
0100c3801458a000_2022-06-19_00-10-17-385

### CODE REVIEW

File: CMakeModules/CopyYuzuQt5Deps.cmake [hunk 1]
```diff
@@ -25,6 +27,8 @@ function(copy_yuzu_Qt5_deps target_dir)
             Qt5Core$<$<CONFIG:Debug>:d>.*
             Qt5Gui$<$<CONFIG:Debug>:d>.*
             Qt5Widgets$<$<CONFIG:Debug>:d>.*
+            Qt5Multimedia$<$<CONFIG:Debug>:d>.*
+            Qt5Network$<$<CONFIG:Debug>:d>.*
```
Comment (german77) [inline CMakeModules/CopyYuzuQt5Deps.cmake:31]: Remove this

File: src/common/input.h [hunk 2]
```diff
@@ -68,6 +77,13 @@ enum class PollingError {
     Unknown,
 };
 
+// Polling mode reply from the controller
```
Comment (german77) [inline src/common/input.h:80]: Wrong comment

**Comment by Docteh** [comment]
I just want to note that Qt's 5.15.2 msvc19 builds do have Qt5Multimedia.dll and Qt5Network.dll from what I can see they're cut from the zip in [ext-windows-bin](https://github.com/yuzu-emu/ext-windows-bin/tree/master/qt)

**Comment by german77** [comment]
It's just a matter of updating the externals. however my Qt build is somehow way bigger than the current one so I don't want to simply increase the Qt size unnecessarily.

**Comment by bunnei** [comment]
Needs a rebase, then this is probably good for merge!

**Review by liamwhite — APPROVED** [review]

**Pull request merged**